docs/index.md

```diff
@@ -62,14 +62,14 @@ for hosting your (binary) artifacts, regardless or your repository layout.
     * [x] HTTP Basic authentication
     * [x] JWT authentication
     * [x] LDAP
-    * [x] Custom authentication provider based on users that resides in second-level cache that exists in OrientDB
+    * [x] Custom authentication provider based on users that reside in the second-level cache that exists in OrientDB
```

Comment:
```
[x] Custom authentication provider, based on users that stored in the second-level cache backed by OrientDB
```

Reply:
[x] Custom authentication provider, based on users that are stored in the second-level cache backed by OrientDB